app/marketing/mails.py

```diff
@@ -105,6 +107,96 @@ def send_mail(from_email, _to_email, subject, body, html=False,
     return response
 
 
+def new_grant(grant, profile):
+    from_email = settings.CONTACT_EMAIL
+    to_email = profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_new_grant_email(grant)
+
+        if not should_suppress_notification_email(to_email, 'new_grant'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
+    finally:
+        translation.activate(cur_language)
+
+
+def new_supporter(grant, subscription):
+    from_email = settings.CONTACT_EMAIL
+    to_email = grant.admin_profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_new_supporter_email(grant, subscription)
+
+        if not should_suppress_notification_email(to_email, 'new_supporter'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
+    finally:
+        translation.activate(cur_language)
+
+
+def thank_you_for_supporting(grant, subscription):
+    from_email = settings.CONTACT_EMAIL
+    to_email = subscription.contributor_profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_thank_you_for_supporting_email(grant, subscription)
+
+        if not should_suppress_notification_email(to_email, 'thank_you_for_supporting'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
+    finally:
+        translation.activate(cur_language)
+
+
+def support_cancellation(grant, subscription):
+    from_email = settings.CONTACT_EMAIL
+    to_email = subscription.contributor_profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_support_cancellation_email(grant, subscription)
+
+        if not should_suppress_notification_email(to_email, 'support_cancellation'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
+    finally:
+        translation.activate(cur_language)
+
+
+def grant_cancellation(grant, subscription):
+    from_email = settings.CONTACT_EMAIL
+    to_email = grant.admin_profile.email
+    cur_language = translation.get_language()
+
+    try:
+        setup_lang(to_email)
+        html, text, subject = render_grant_cancellation_email(grant, subscription)
+
+        if not should_suppress_notification_email(to_email, 'grant_cancellation'):
+            send_mail(from_email, to_email, subject, text, html, categories=['transactional', func_name()])
+    finally:
+        translation.activate(cur_language)
+
+
+def subscription_terminated(grant, subscription):
```

Comment:
which of these emails still need to be WIRED up still?  worth throwing a TODO in here